### PR TITLE
MRG: Add on_missing argument to EpochsArray

### PR DESCRIPTION
**Pull request by cmmoenne**
PR to address issue #4041.

It's very simple, I just added on_missing to EpochsArray class.

### CODE REVIEW

File: mne/epochs.py [hunk 1]
```diff
@@ -1987,6 +1987,8 @@ class EpochsArray(BaseEpochs):
         a <= t <= b.
     proj : bool | 'delayed'
         Apply SSP projection vectors. See :class:`mne.Epochs` for details.
+    on_missing : str
+        See `Epochs` docstring.
```
Comment (Eric89GXL) [inline mne/epochs.py]:
```
:class:`mne.Epochs`
```

Comment (cmmoenne) [inline mne/epochs.py]: updated!

**Comment by Eric89GXL** [comment]
Thanks for the quick fix. I don't think this needs a test because it's really a function of the base class.

Is this ready for merge from your end? If so please add `MRG:` to the title to remind people to review and comment

**Comment by agramfort** [comment]
+1 for merge when CIs are happy

**Comment by Eric89GXL** [comment]
Thanks @cmmoenne

**Pull request merged**